### PR TITLE
[14.0][MIG] passport expiration

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

File: passport_expiration/hr_view.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_employee_form_inherit_passport_expiration" model="ir.ui.view">
```
Comment (i-vyshnevska) [inline passport_expiration/hr_view.xml]:
```suggestion
    <record id="view_employee_form" model="ir.ui.view">
```

File: passport_expiration/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Passport Expiration",
+    "version": "14.0.0.1.0",
+    "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
+    "maintainer": "Savoir-faire Linux",
+    "website": "https://github.com/OCA/vertical-travel",
+    "license": "AGPL-3",
+    "category": "",
```
Comment (i-vyshnevska) [inline passport_expiration/__manifest__.py]:
```suggestion
```

File: passport_expiration/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from . import hr
```
Comment (i-vyshnevska) [inline passport_expiration/__init__.py]:
```suggestion
from . import models
```

File: passport_expiration/i18n/fr.po [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+# 	* hr_employee_phone_extension
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
```
Comment (StephaneMangin) [inline passport_expiration/i18n/fr.po]:
```suggestion
"Project-Id-Version: Odoo Server 14.0\n"
```

File: passport_expiration/i18n/passport_expiration.pot [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Translation of OpenERP Server.
```
Comment (i-vyshnevska) [inline passport_expiration/i18n/passport_expiration.pot]: this file can be dropped

File: passport_expiration/i18n/fr.po (hunk 4 above)
Comment (StephaneMangin) [reply passport_expiration/i18n/fr.po]: Maybe you should fully regenerate the translations ?

**Review by i-vyshnevska — APPROVED** [review]
LG some nitpicking

**Review by StephaneMangin — APPROVED** [review]
LG until translation fit the Odoo version

File: passport_expiration/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2013 Savoir-faire Linux
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Passport Expiration",
+    "version": "14.0.0.1.0",
+    "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
+    "maintainer": "Savoir-faire Linux",
```
Comment (ivantodorovich) [inline passport_expiration/__manifest__.py]:
```suggestion
```
This key is not used afaik

File: passport_expiration/models/hr.py [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# Copyright 2013 Savoir-faire Linux
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+    passport_expiration_date = fields.Date(
```
Comment (ivantodorovich) [inline passport_expiration/models/hr.py:10]:
```suggestion

    passport_expiration_date = fields.Date(
```

**Comment by ivantodorovich** [comment]
Isn't it worth moving this module https://github.com/OCA/hr ?

At first glance it seems that's a better repo for this module.
The module should be renamed to `hr_employee_passport_expiration_date`, `*_expire_date` or similar

**Comment by damdam-s** [comment]
@OCA/vertical-travel-maintainers do you mind if I move this module to oca/hr repo?

@ivantodorovich that was my initial idea and then I saw this module existed in this repo. BTW, no move since ages in this repo so let's wait the maintainers answer.
if no answers, I will move it to oca/hr by the end of the month

**Pull request merged**